### DISCUSSION TITLE
use MediatR to add slash commands

### DISCUSSION THREAD

**Discussion by DevilHimSelf666** [discord-net/Discord.Net#2432] [Q&A]
Hi, i was working with MediatR for the bot and now I need to add a slash command.

I don't know the best way to add a slash command. I can inject an InteractionService to my DiscordEventListener but it does not seem right.

is anyone have any suggestions about it?

**Comment by dukesteen** [comment]
Hi devil, if you join the discord we can help you out more efficiently: https://discord.gg/dnet
